app/configs/config.py

```diff
@@ -69,23 +69,23 @@ def url_file_tag(self) -> str:
 
     @computed_field
     def url_upload_greenroom(self) -> str:
```

Comment:
perhaps we could change to `url_fileops_greenroom` same for core as well?

Reply:
ahhh thanks for catching that! will update them to `fileops`